core/tests/wdio.conf.js

```diff
@@ -15,8 +15,10 @@ var chromeVersion = (args[0] == 'DEBUG=true') ? args[6] : args[5];
 var chromedriverPath =
 './node_modules/webdriver-manager/selenium/chromedriver_' + chromeVersion;
 
-var spw = null;
-var vidPath = '';
+// If video recorder is not running the ffmpeg process will be null.
+var ffmpegProcess = null;
+// The absolute path where the recorded video of test will be stored.
+var videoPath = '';
```

Comment:
If this is an absolute path then shouldn't it start with a '/'?

If it is only set later then why are we defaulting to '' instead of null?

Reply:
Initialised with null. We are using directly getting the current directory path PTAL [here](https://github.com/oppia/oppia/blob/develop/core/tests/protractor.conf.js#L318) using path library, so following the same pattern in wdio file as well.